lib/publiccloud/provider.pm

```diff
@@ -522,7 +523,9 @@ sub terraform_apply {
     if (get_var('PUBLIC_CLOUD_NVIDIA')) {
         $cmd .= "-var gpu=true ";
     }
-    $cmd .= "-var 'ssh_public_key=" . $self->ssh_key . ".pub' ";
+    unless (is_openstack) {
+        $cmd .= "-var 'ssh_public_key=" . $self->ssh_key . ".pub' ";
```

Comment:
hm , dropping this won't actually change the fact that we still have variable in tf file https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/publiccloud/terraform/openstack.tf#L29 so it will defaults to empty string instead . Not sure what would actually happen here in such case https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/publiccloud/terraform/openstack.tf#L81 anyway I would align openstack.tf with that change ...

Reply:
that is keypair, different variable. Using this new one was causing a failure as it was not introduced before.

Reply:
`-var 'keypair=openqa1709640672_0'` is correctly populated -> http://kepler.suse.cz/tests/22699#step/prepare_openstack/154